src/internal/snapshot/util.go

```diff
@@ -0,0 +1,29 @@
+package snapshot
+
+import (
+	"github.com/pachyderm/pachyderm/v2/src/internal/clusterstate"
+	"github.com/pachyderm/pachyderm/v2/src/internal/dockertestenv"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage/chunk"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage/fileset"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage/kv"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage/track"
+	"google.golang.org/grpc"
+	"testing"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/grpcutil"
+	"github.com/pachyderm/pachyderm/v2/src/snapshot"
+)
+
+func NewTestClient(t testing.TB) snapshot.APIClient {
```

Comment:
This probably doesn't need to be used outside of this package ever, so it should be private.